app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1
+
+DESCRIPTION="Summarizes the contents of a syslog log file"
+HOMEPAGE="https://github.com/dpaleino/syslog-summary"
+SRC_URI="mirror://github/dpaleino/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
Could you please verify the license? Look at the copyright notice on top of source files. If it happens to state '[GPL] version 3 or (at your blah blah) newer', then it should be `GPL-3+` instead.

Reply:
thanks for spotting that @mgorny - now fixed.